pkg/daemon/ceph/agent/flexvolume/server.go

```diff
@@ -48,6 +49,15 @@ const (
 
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "flexvolume")
 
+// getRookFlexLocation returns the path of rook flex volume driver
+func getRookFlexLocation() string {
```

Comment:
I think this function is well suited for a `util.go` in this package.

Reply:
Cool will move it

Reply:
Is the function name ok or should i change it ?

Reply:
`getRookFlexBinaryPath`?

Reply:
yeah thats good

Reply:
made the changed as requested